### ISSUE TITLE
Plot: a way to add custom CSS animation?

### ISSUE THREAD

**Issue by h4mzt3r** [neveldo/jQuery-Mapael#269]
Hi guys, 

Is there a way to add custom CSS animation on a plot? I'm interrested to add a pulse effect (like http://codepen.io/olam/pen/zcqea) on my plots, but I see no options/attributes to add a class. 

Thanks for your help.

**Comment by neveldo** [comment]
Hi @h4mzt3r . For now, there is no option to add custom CSS class to areas or plotted points. If you have the time, feel free to make a pull-request to add the option, I think it could be a useful feature ! (otherwise, I will add it to my todos)

However, this is not required for your needs because you still can apply custom CSS to your areas or plots by using the proper CSS selector, for instance, to set styles to a plot with its ID :

```
.map circle[data-id="your-plot-id"] {
    /* styles */
}
```

... Or to all plots of the map : 

```
.map circle {
    /* styles */
}
```

By this way, you should be able to set your custom animation on the wanted plots within your map.

**Comment by Indigo744** [comment]
Custom CSS class is available with Mapael 2.1.0 (https://github.com/neveldo/jQuery-Mapael/releases/tag/2.1.0).
